lib/wda/webdriveragent.js

```diff
@@ -226,7 +226,7 @@ class WebDriverAgent {
       // make sure that the WDA dependencies have been built
       const synchronizationKey = path.normalize(this.bootstrapPath);
       await SHARED_RESOURCES_GUARD.acquire(synchronizationKey, async () => {
-        const didPerformUpgrade = await checkForDependencies(this.bootstrapPath, this.useCarthageSsl);
+        const didPerformUpgrade = await checkForDependencies({useSsl: this.useCarthageSsl});
```

Comment:
do we also need to set the minimum component version in package.json?

Reply:
No, since this repository already has the apppium-webdriveragent as `^0.1.0`
(this change happened before 0.1.0)